.eslintrc.json

```diff
@@ -153,5 +153,13 @@
     "docs/sidebars.ts",
     "docs/src/**",
     "docs/blog/**"
+  ],
+  "overrides": [
+    {
+      "files": ["*.js"],
+      "rules": {
+        "@typescript-eslint/explicit-function-return-type": "off"
```

Comment:
1. Add the return type to the function of the offending file.
2. This is not a solution.